### PR TITLE
[ENH] Second-level analysis

### PR DESCRIPTION
**Pull request by rciric**
* Implement second-level analysis workflow for the example dataset.
* Add a nipype interface for `ptoz`.

### CODE REVIEW

File: workflows.py [hunk 1]
```diff
@@ -135,8 +136,12 @@ def second_level_wf(output_dir, name='wf_2nd_level'):
 
     # Thresholding - FWE ################################################
     # smoothest -r %s -d %i -m %s
+    fwe_smoothness = pe.Node(fsl.SmoothEstimate(), name='fwe_smoothness')
     # ptoz 0.05 -g %f
+    fwe_ptoz = pe.Node(PtoZ(pvalue=0.05), name='fwe_ptoz')
     # fslmaths %s -thr %s zstat1_thresh
+    fwe_thresh = pe.Node(fsl.Threshold(), name='fwe_thresh',
+                            iterfield=['in_file', 'thresh'])
```
Comment (oesteban) [inline workflows.py]:
```suggestion
                         iterfield=['in_file', 'thresh'])
```

**Comment by satra** [comment]
@rciric - these workflows may be of help:

1. model estimation run level: https://github.com/gablab/openfmri/blob/openfmri/subject_level/fmri_ants_openfmri.py#L810
2. fixedfx workflow (combining runs): https://github.com/gablab/openfmri/blob/openfmri/subject_level/fmri_ants_openfmri.py#L864

the core of the specific workflows are defined here:
https://github.com/gablab/openfmri/blob/openfmri/subject_level/fmri_ants_openfmri.py#L32

3. group level stats: https://github.com/gablab/openfmri/blob/openfmri/group_level/group_multregress_openfmri.py

we do also have additional versions with code that detects which subjects have estimable contrasts (due to missing conditions) at the run level and how these are combined at the fixed effects level.

File: interfaces.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+class PtoZ(FSLCommand):
+    """Determine the z-value corresponding to an observed p-value."""
+    input_spec = PtoZInputSpec
+    output_spec = PtoZOutputSpec
+    _cmd = 'ptoz'
+
+    def aggregate_outputs(self, runtime=None, needed_outputs=None):
+        outputs = self._outputs()
+        outputs.zstat = float(runtime.stdout)
```
Comment (oesteban) [inline interfaces.py]: Possibly necessary (please check):
```suggestion
        outputs.zstat = float(runtime.stdout.strip())
```

**Review by oesteban — COMMENTED** [review]
Ref @satra's comments - we need something clear that works well for this case. It is not the intent of this code to be able to generalize to other contrasts / modeling options. Although I agree that those links you provided could be written somewhere for users to adapt the workflow to their needs.

**Comment by satra** [comment]
@oesteban - i completely agree that this should be focused on this analysis. i only posted it to see if those existing workflows could directly help in some cases.

**Comment by oesteban** [comment]
Thanks, sorry for being overprotective about @rciric's attention :P

**Comment by rciric** [comment]
Thanks for looking at this @oesteban ! And thanks for the link @satra -- it turned out to be a very helpful reference as well!

Is there a dataset that I can test on? Perhaps `/oak/stanford/groups/russpold/data/openfmri/derivatives/ds000003/fmriprep_1.3.2/fmriprep/`? Anywhere that the first-level has been run? I'm not sure that `fwe_thresh` needs to be a `MapNode` at all, so I've switched it to a regular `Node` for now.

**Comment by oesteban** [comment]
is this finished?

**Comment by rciric** [comment]
Should be done/ready for review now -- just removed the `[WIP]` tag.

**Pull request merged**